tests/test_smartcar.py

```diff
@@ -189,7 +189,30 @@ def test_get_auth_url_vehicle_info_dictionary(self):
         })
 
         expected = smartcar.const.CONNECT_URL + '/oauth/authorize?' + query
-        
+
+        expected_params = parse_qs(expected)
+        actual_params = parse_qs(actual)
+
+        assertDeepEquals(self, expected_params, actual_params)
+
+    def test_get_auth_url_single_select(self):
```

Comment:
Can also add tests for junk info being passed in like you did for the javascript sdk!

Reply:
Good idea! Even found a small bug doing so. Fixed.